sound/soc/sof/control.c

```diff
@@ -51,29 +51,38 @@ int snd_sof_volume_get(struct snd_kcontrol *kcontrol,
 	unsigned int i, channels = scontrol->num_channels;
 	int err, ret;
 
-	ret = pm_runtime_get_sync(sdev->dev);
-	if (ret < 0) {
-		dev_err(sdev->dev, "error: volume get failed to resume %d\n",
-			ret);
-		return ret;
+	if (!scontrol->is_valid) {
```

Comment:
what does is_valid mean here ? do you mean is_dsp_resident instead. Please also comment this conditional.